bdk_core_example/src/main.rs

```diff
@@ -189,29 +190,37 @@ fn main() -> anyhow::Result<()> {
             }
         }
         Commands::Balance => {
-            let (confirmed, unconfirmed) = tracker.iter_unspent_full(&chain).fold(
-                (0, 0),
-                |(confirmed, unconfirmed), (spk_index, utxo)| {
-                    if utxo.confirmed_at.is_some() || spk_index.0 == Keychain::Internal {
-                        (confirmed + utxo.value, unconfirmed)
+            // tracker.iter_unspent(&chain, &graph)
+            //     .filter_map(|(spk_i, op)| chain.full_txout(graph, op));
+            let (confirmed, unconfirmed) = tracker
+                .iter_unspent(&chain, &graph)
+                .filter_map(|(spk_i, op)| chain.full_txout(&graph, op).map(|utxo| (spk_i, utxo)))
+                .fold((0, 0), |(confirmed, unconfirmed), (spk_index, utxo)| {
+                    if utxo.height.is_confirmed() || spk_index.0 == Keychain::Internal {
+                        (confirmed + utxo.txout.value, unconfirmed)
                     } else {
-                        (confirmed, unconfirmed + utxo.value)
+                        (confirmed, unconfirmed + utxo.txout.value)
                     }
-                },
-            );
+                });
 
             println!("confirmed: {}", confirmed);
             println!("unconfirmed: {}", unconfirmed);
         }
         Commands::Txo { utxo_cmd } => match utxo_cmd {
             TxoCmd::List => {
-                for (spk_index, txout) in tracker.iter_txout_full(&chain) {
+                for (spk_index, txout) in
+                    tracker
+                        .iter_unspent(&chain, &graph)
```

Comment:
This now only iters unspent. It's meant to iter all txouts in the chain. Should be `iter_txout` instead.